mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,8 @@
     <!-- Content description (not visible, for screen readers etc.): Un-bookmark the current page -->
     <string name="browser_menu_edit_bookmark">Edit bookmark</string>
     <!-- Browser menu button that sends a user to help articles -->
+    <string name="browser_menu_addon_manager">Add-ons Manager</string>
```

Comment:
I think the comment is wrong here, as it doesn't correspond to the string (also, it's the same as the one below ;) )